### ISSUE TITLE
File uploading get stuck at `transfer`

### ISSUE THREAD

**Issue by tete1030** [liximomo/vscode-sftp#316]
## Expected Behavior
File should be uploaded and produce `upload done` in log.

## Actual Behavior
File is not uploaded and there is only `transfer xxx` in log.

## Steps to Reproduce the Problem

  1. Right click on a file in `Explorer` panel, click `Upload`

or 

  1. When `uploadOnSave` is enabled, modify a file and save it

### Extension Logs

```
[info] config at /Users/user/Desktop/docs/my {"remotePath":"./","uploadOnSave":true,"downloadOnOpen":false,"syncMode":"update","ignore":[".vscode",".git",".DS_Store",".sublime-project",".sublime-workspace","venv/",".svn/",".hg/",".git/",".bzr","_darcs","CVS","Thumbs.db","desktop.ini","data/","*.swp","*.pyc","*.so","*.o"],"concurrency":4,"protocol":"sftp","connectTimeout":10000,"interactiveAuth":false,"secure":false,"passive":false,"port":22,"profiles":{"server1":{"host":"xxx.xxx.xxx.1","port":22,"username":"xxx","privateKeyPath":"xxx","remotePath":"/home/user/my/docs"},"server2":{"host":"xxx.xxx.xxx.2","port":22,"username":"xxx","privateKeyPath":"xxx","remotePath":"/home/user/my/docs"},"server3":{"host":"xxx.xxx.xxx.3","port":22,"username":"xxx","privateKeyPath":"xxx","remotePath":"/home/user/docs"},"server4":{"host":"xxx.xxx.xxx.4","port":22,"username":"xxx","privateKeyPath":"xxx","remotePath":"/home/user/docs"}},"watcher":{"files":null,"autoUpload":false,"autoDelete":false}}
[trace] run command 'Set Profile'
[trace] run command 'Upload File'
[trace] handle upload file for /Users/user/Desktop/docs/my/docs/file.py
[trace] transfer from /Users/user/Desktop/docs/my/docs/file.py to docs/file.py
[info] transfer /Users/user/Desktop/docs/my/docs/file.py
[info] [file-save] /Users/user/Desktop/docs/my/docs/file.py
[trace] handle upload file for /Users/user/Desktop/docs/my/docs/file.py
[trace] transfer from /Users/user/Desktop/docs/my/docs/file.py to docs/file.py
[info] transfer /Users/user/Desktop/docs/my/docs/file.py
```

## Specifications

  - Version: 1.5.4
  - Platform: macOS 10.13.6, VSCode 1.27.2

The network and configurations of these servers should be right because It previously worked until some point, maybe when I upgraded VSCode or this sftp extension is automatically upgraded after I restart VSCode. **Besides, the "SFTP: REMOTE EXPLORER" panel works correctly.**

**Comment by liximomo** [comment]
Fix in v1.5.5.

**Comment by tete1030** [comment]
The only difference I see in v1.5.5 is that it now reports "done" in the status bar instead of stoping at "transfer". However the file is still not uploaded successfully. And in sftp output panel, it still does not report any success regarding the uploading.

**Comment by tete1030** [comment]
I found that the files were uploaded to home directory as in #317 . The `remotePath` config is not honored.

**Comment by liximomo** [comment]
Ok. This can be closed.

**Comment by liximomo** [comment]
@tete1030 This is fixed in v1.5.6.
